src/CMakeLists.txt

```diff
@@ -396,6 +403,17 @@ foreach (f IN LISTS HTML_TEMPLATE_FILES)
     target_sources(Halide_c_templates PRIVATE ${DST})
 endforeach ()
 
+##
+# Build the serialization header file.
+##
+add_library(Halide_serialization_headers OBJECT)
+add_custom_command(OUTPUT "$<SHELL_PATH:${CMAKE_CURRENT_SOURCE_DIR}/halide_ir_generated.h>"
+                   COMMAND flatc --cpp -o "$<SHELL_PATH:${CMAKE_CURRENT_SOURCE_DIR}>" "$<SHELL_PATH:${CMAKE_CURRENT_SOURCE_DIR}/halide_ir.fbs>"
```

Comment:
If `flatc` is a target, then this is fine, but I'll need to look at the flatbuffers documentation to be sure. It would be a lot more typical to have `flatbuffers::flatc` or the like.